diagnostics-app/src/app/layout/side-nav/side-nav.component.css

```diff
@@ -4,11 +4,28 @@
  * found in the LICENSE file.
  */
 
-:host {
+ :host {
   background: inherit;
-}
+ }
 
-.mat-subheader {
+ .mat-subheader {
```

Comment:
Is there a reason behind having these whitespace?

Reply:
As our discussion before, I'll do the style change in another PR.

Reply:
SGTM.